UPGRADING.md

```diff
@@ -127,6 +127,45 @@ By default, the new `MinInitialDepositRatio` parameter is set to zero during mig
 feature is disabled. If chains wish to utilize the minimum proposal deposits at time of submission, the migration logic needs to be 
 modified to set the new parameter to the desired value.
 
+##### New Proposal.Proposer field
```

Comment:
@tac0turtle, given we backported the addition of the proposer field, we may want to partly backport this migration too, wdyt?

Reply:
yea we can do that once this is merged.